### ISSUE TITLE
readConfObject not working when conf is an array

### ISSUE THREAD

**Issue by garrettjstevens** [GMOD/jbrowse-components#60]
@rbuels I think I'm trying to use `readConfObject()` correctly, but let me know if I'm not. I can do this just fine:
```js
readConfObject(MODEL.configuration.tracks[0], 'name')
// "volvox-sorted red/blue"
```
but this doesn't seem to work:
```js
readConfObject(MODEL.configuration.tracks[0], 'category')
// Proxy {0: ScalarNode$$1, 1: ScalarNode$$1, length: 2, $treenode: ObjectNode$$1, toJSON: ƒ, Symbol(mobx administration): ObservableArrayAdministration}
```
If you do a `getSnapshot()` it works, though:
```js
getSnapshot(readConfObject(MODEL.configuration.tracks[0], 'category'))
// ["Bar Category", "Baz Category"]
```
I've tried to start fixing it, but got a bit confused and thought I'd let you know in case you have a quick solution.

**Comment by rbuels** [comment]
oh hmm.  yes i think readconfobject should probably do a `getSnapshot` on the value it returns if it is a MST node (`isStateTree()` i think).  does that work?

**Comment by garrettjstevens** [comment]
Yep, seems to do the trick. Thanks. I'll PR the change in a bit.

**Comment by garrettjstevens** [comment]
Was fixed in #64
